internal/service/credentials/resource_credential_issuer_profile_test.go

```diff
@@ -0,0 +1,199 @@
+			resource.TestMatchResourceAttr(resourceFullName, "id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "environment_id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "application_instance_id", verify.P1ResourceIDRegexp),
+			resource.TestCheckResourceAttrSet(resourceFullName, "created_at"),
+			resource.TestCheckResourceAttrSet(resourceFullName, "updated_at"),
+			resource.TestCheckResourceAttr(resourceFullName, "name", updatedName),
+		),
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheckEnvironment(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		CheckDestroy:             nil, // Note: Issuer Profiles aren't deleted once created. Placeholder if this changes.  testAccCheckCredentialIssuerProfileDestroy,
+		ErrorCheck:               acctest.ErrorCheck(t),
+		Steps: []resource.TestStep{
+			// initial profile
+			initialProfile,
+			{
+				Config:  testAccCredentialIssuerProfile_Full(resourceName, name),
+				Destroy: true,
+			},
+			// update profile
+			updatedProfile,
+			{
+				Config:  testAccCredentialIssuerProfile_Full(resourceName, updatedName),
+				Destroy: true,
+			},
+			// changes
+			initialProfile,
+			updatedProfile,
+			initialProfile,
+			{
+				Config:  testAccCredentialIssuerProfile_Full(resourceName, name),
+				Destroy: true,
+			},
+			{
+				Config:  testAccCredentialIssuerProfile_Full(resourceName, updatedName),
+				Destroy: true,
+			},
+		},
+	})
+}
+
+func TestAccCredentialIssuerProfile_InvalidConfig(t *testing.T) {
+	t.Parallel()
+
+	resourceName := acctest.ResourceNameGen()
+	name := acctest.ResourceNameGen()
+
+	environmentName := acctest.ResourceNameGenEnvironment()
+	licenseID := os.Getenv("PINGONE_LICENSE_ID")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheckEnvironment(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		CheckDestroy:             nil,
+		ErrorCheck:               acctest.ErrorCheck(t),
+		Steps: []resource.TestStep{
+			{
+				Config:      testAccCredentialIssuerProfileInvalidConfig_InvalidName(resourceName, name),
+				ExpectError: regexp.MustCompile("Error: Error when calling `CreateCredentialIssuerProfile`: Validation Error : \\[name must not be empty or blank\\]"),
+			},
+			{
+				Config:      testAccCredentialIssuerProfileInvalidConfig_CredentialServuceNotEnabled(environmentName, licenseID, resourceName, name),
+				ExpectError: regexp.MustCompile("Error: Error when calling `ReadCredentialIssuerProfile`: Issuer not found for environment"),
+			},
+		},
+	})
+}
+
+func testAccCredentialIssuerProfile_Full(resourceName, name string) string {
+	return fmt.Sprintf(`
+	%[1]s
+resource "pingone_credential_issuer_profile" "%[2]s" {
+  environment_id = data.pingone_environment.credentials_test.id
+  name           = "%[3]s"
+}`, acctest.CredentialsSandboxEnvironment(), resourceName, name)
```

Comment:
Can this be changed to use `acctest.MinimalSandboxEnvironment` so we're creating an environment each time?  This will test the API race conditions

Reply:
updated. testing.